packages/ember-glimmer/tests/integration/syntax/with-test.js

```diff
@@ -256,35 +244,6 @@ moduleFor('Syntax test: Multiple {{#with as}} helpers', class extends RenderingT
     this.assertText('caterpillar');
   }
 
-  ['@test nested {{with}} blocks shadow the outer scoped variable properly.']() {
-    this.render(`{{#with first as |ring|}}{{ring}}-{{#with fifth as |ring|}}{{ring}}-{{#with ninth as |ring|}}{{ring}}-{{/with}}{{ring}}-{{/with}}{{ring}}{{/with}}`, {
-      first: 'Limbo',
-      fifth: 'Wrath',
-      ninth: 'Treachery'
-    });
-
-    this.assertText('Limbo-Wrath-Treachery-Wrath-Limbo');
-
-    this.runTask(() => this.rerender());
-
-    this.assertText('Limbo-Wrath-Treachery-Wrath-Limbo');
-
-    this.runTask(() => {
-      set(this.context, 'first', 'I');
-      set(this.context, 'ninth', 'K');
-    });
-
-    this.assertText('I-Wrath-K-Wrath-I');
-
-    this.runTask(() => {
-      set(this.context, 'first', 'Limbo');
-      set(this.context, 'fifth', 'Wrath');
-      set(this.context, 'ninth', 'Treachery');
-    });
-
-    this.assertText('Limbo-Wrath-Treachery-Wrath-Limbo');
-  }
-
```

Comment:
Duplicate of "re-using the same variable with different {{#with}} blocks does not override each other"